storage/umass/scsi.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Phoenix-RTOS
+ *
+ * USB Mass Storage class driver
+ *
+ * SCSI transparent command set definitions header
+ *
+ * Copyright 2024 Phoenix Systems
+ * Author: Adam Greloch
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef _SCSI_H_
+#define _SCSI_H_
+
+#include <unistd.h>
+
+#define CBW_SIG 0x43425355
+#define CSW_SIG 0x53425355
+
+
+/* Opcodes */
+#define SCSI_REQUEST_SENSE 0x03
+#define SCSI_INQUIRY       0x12
+
+
+typedef struct {
+	uint32_t sig;
+	uint32_t tag;
+	uint32_t dlen;
+	uint8_t flags;
+	uint8_t lun;
+	uint8_t clen;
+	uint8_t cmd[16];
+} __attribute__((packed)) umass_cbw_t;
```

Comment:
Is this umass- or scsi-specific? Or a specific umass container for scsi data?

Reply:
It is a umass-specific container for scsi commands. Yeah, scsi.h is not the best place for it